Formula/telnet.rb

```diff
@@ -0,0 +1,37 @@
+class Telnet < Formula
+  desc "Apple telnet built from macOS 10.12.4 sources"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
+  sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
+
+  depends_on :xcode => :build
```

Comment:
I am pretty sure that any more or less modern Xcode version would suffice:

- Sources are practically unchanged after 2008.
- `.xcodeproj` of `libtelnet` does not seem to have any outstanding build settings.
- `telnet.tproj/Makefile` also does not contain anything suspicious.

So yeah, I think we are pretty safe to assume that Xcode version does not make much of a difference.